libbpf-cargo/src/build.rs

```diff
@@ -243,31 +246,31 @@ fn compile(
     clang: &Path,
     mut clang_args: Vec<OsString>,
     target_dir: &Path,
-) -> Result<()> {
+) -> Result<Vec<Vec<u8>>> {
```

Comment:
Shouldn't `build` be adjusted to bubble up this value as well?

Reply:
I didn't modify `build::build`. It is only used in the binary crate and it's not obvious how to report these errors in the binary crate - the data is available in `build` for each file if someone decides how it should be displayed.